batchflow/config.py

```diff
@@ -1,250 +1,257 @@
 """ Config class"""
-from pprint import pformat
-import numpy as np
-
-
-class Config:
-    """ Class for configs that can be represented as nested dicts with easy indexing by slashes """
-
-    class IAddDict(dict):
-        """ dict that supports update via += """
-        def __iadd__(self, other):
-            if isinstance(other, dict):
-                self.update(other)
-            else:
-                raise TypeError(f"unsupported operand type(s) for +=: 'IAddDict' and '{type(other)}'")
-            return self
 
+class Config(dict):
```

Comment:
Reorder methods in more structured way:
- init (`__init__`, `parse`)
- methods for elements (`put`, `get`, `pop`, `_get`)
- dict methods (`update`, `flatten`, `copy`, `keys`, `values`, `items`)
- content dunder methods (`__getitem__`, `__getitem__`, `__getattr__`, `__delitem__`)
- arithmetic dunder methods (`__iadd__`, ...)
- pickle dunder (`__getstate__`, `__setstate__`)